README.md

```diff
@@ -36,49 +36,25 @@ Run the application:
 $ oneview-redfish-toolkit
 ```
 
- At first time, it will create all the needed configuration files under user directory, and will prompt for the OneView IP(s).
+At first time, it will create all the needed configuration files under user directory, and will prompt for the OneView IP(s) to customize `redfish.conf` file with the entered IP. The following files will be created:
 
-You can update the configuration files created under the user directory, or if you want to use custom configuration files you can pass them as arguments:
+* $HOME/.config/oneview-redfish-toolkit/
+  * redfish.conf
+  * logging.conf
+  * redfish.log
+  * redfish_performance.log
+  * redfish_ov_data.log
+  * certs/
 
-```bash
-$ oneview-redfish-toolkit --config redfish.conf --log-config logging.conf
-```
+The `redfish.conf` and `logging.conf` are used for toolkit customization and certs directory is used to place SCMB certificates retrieved from OneView instances when Event Service is enabled.
 
-### Development Environment
-
-We recommend to run inside a virtual environment. You can create one running:
+You can customize the configuration files created under the user directory, or if you want to use your own custom configuration files you can pass them as arguments. If no arguments are passed the application will use the ones on user directory:
 
 ```bash
-$ virtualenv env_name_you_choose -p python3.5 # to create a Python3.5 environment, for example
-$ source env_name_you_choose/bin/activate # load the environment
-```
-
-Once the environment is loaded, download and uncompress the latest version from [releases page](https://github.com/HewlettPackard/oneview-redfish-toolkit/releases), or clone current development version running:
-
-```bash
-$ git clone https://github.com/HewlettPackard/oneview-redfish-toolkit.git
-```
-
-Then, proceed with:
-
-```bash
-$ cd oneview-redfish-toolkit # enter the service folder
-# edit redfish.conf
-$ pip install -r requirements.txt # to install all requirements into the virtual environment
-$ ./run.sh    # to launch the service
+$ oneview-redfish-toolkit --config redfish.conf --log-config logging.conf
 ```
 
-## SDK Documentation
-
-The latest version of the SDK documentation can be found in the [SDK Documentation section](https://hewlettpackard.github.io/oneview-redfish-toolkit/index.html).
-
-> Note: This documentation has been manually updated following the steps found [here](https://github.com/HewlettPackard/python-hpOneView/blob/master/deploy.sh).
-
-## Logging
-
-Logging configuration can be found in `logging.conf` file. The provided configuration enables INFO level at both console and file output (which will generate a `redfish.log` file).
-
-## Configuration
+## Toolkit Configuration
 
 In order to start up oneview-redfish-toolkit service, there is some mandatory configuration at `redfish.conf` file to provide as explained below:
```

Comment:
What is it that is mandatory?  Does the user have to do anything here?  Or ... it is there and must stay there?
This sounds like the user must change something in order to get it to run.

Reply:
Sorry, there is the OneView credentials. Maybe we could pull out a section just to fill these properties.

Reply:
Done.